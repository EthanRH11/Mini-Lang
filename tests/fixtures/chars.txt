begin:

    char c = 'z';
    char d = 'a';

    out_to_console(d + " is the first letter of the alphabet."); ...
    out_to_console(c + " is the last letter of the alphabet.");

end