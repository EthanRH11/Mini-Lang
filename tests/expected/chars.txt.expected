a is the first letter of the alphabet.    
z is the last letter of the alphabet.